<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dodawanie</class>
 <widget class="QDialog" name="Dodawanie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodawanie</string>
  </property>
  <widget class="QLineEdit" name="pTytul">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pAutor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pRokWydania">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pGatunek">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lTytul">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>32</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tytu≈Ç:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lAutor">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>72</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Autor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lRokWydania">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rok wydania:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lGatunek">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>152</y>
     <width>47</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gatunek:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dodajButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
